tools/make_patch.c

```diff
@@ -108,9 +108,21 @@ int create_symbol(struct symbol **tip, const int value, const char *name)
 const struct symbol *find_symbol(const struct symbol *symbols, const char *name)
 {
 	const struct symbol *symbol = symbols;
+	int namelen = strlen(name);
+	int symbolnamelen = 0;
+	const char *equalLastCharacters = NULL;
```

Comment:
Watch the styling of the entire file, don't introduce inconsistent styles of variable names...

Reply:
equalLastCharacters was removed in https://github.com/pret/pokecrystal/pull/882/commits/74142c471fc03d09c01ea35bfd59952b69b612c1

Reply:
renamed lastFoundSymbol to currentpatch in https://github.com/pret/pokecrystal/pull/882/commits/bf8178e7328da3240535ef9800cdb5163466bcde

Reply:
This should be resolved now?

Reply:
`current_patch` would be more consistent.

Reply:
Renamed to current_patch in https://github.com/pret/pokecrystal/pull/882/commits/1cbd3f1712b5a18f07fcbd8032b78d68e19bfb59